automation/taskcluster/rename_snapshot_artifacts.py

```diff
@@ -76,6 +76,8 @@ def does_file_name_contain_timestamp(filename):
 
     for extension, hash_extension in itertools.product(ARTIFACT_EXTENSIONS, HASH_EXTENSIONS):
         if filename.endswith(extension + hash_extension):
+            if "-unspecified" in filename:
+                return False
```

Comment:
I've added this here, because for reasons that are unclear to me, `:publish` generates two sets of artifacts, one of which as `-unspecified` as version.  We don't need them, but I'm not sure how to not generate them in the first place:

```
components/tooling/glean-gradle-plugin/build/maven
├── android-components
│   └── tooling-glean-gradle
│       ├── maven-metadata.xml
│       ├── maven-metadata.xml.md5
│       ├── maven-metadata.xml.sha1
│       └── unspecified
│           ├── tooling-glean-gradle-unspecified.jar
│           ├── tooling-glean-gradle-unspecified.jar.md5
│           ├── tooling-glean-gradle-unspecified.jar.sha1
│           ├── tooling-glean-gradle-unspecified.pom
│           ├── tooling-glean-gradle-unspecified.pom.md5
│           └── tooling-glean-gradle-unspecified.pom.sha1
└── org
    └── mozilla
        └── components
            └── tooling-glean-gradle
                ├── 28.0.0-SNAPSHOT
                │   ├── maven-metadata.xml
                │   ├── maven-metadata.xml.md5
                │   ├── maven-metadata.xml.sha1
                │   ├── tooling-glean-gradle-28.0.0-20200109.133335-1.jar
                │   ├── tooling-glean-gradle-28.0.0-20200109.133335-1.jar.md5
                │   ├── tooling-glean-gradle-28.0.0-20200109.133335-1.jar.sha1
                │   ├── tooling-glean-gradle-28.0.0-20200109.133335-1.pom
                │   ├── tooling-glean-gradle-28.0.0-20200109.133335-1.pom.md5
                │   └── tooling-glean-gradle-28.0.0-20200109.133335-1.pom.sha1
                ├── maven-metadata.xml
                ├── maven-metadata.xml.md5
                └── maven-metadata.xml.sha1
```